app/views/contacts/_search_fields.html.slim

```diff
@@ -0,0 +1,25 @@
+= link_to t('common.contacts.add_or_edit_address'), contacts_url, target: '_blank'
+br
+br
+form data-remote="true" action="/contacts_search" role="search"
+  fieldset
+    input#popup-search.form-control type="text" name="contacts_search_value" 
```

Comment:
One thing I was going to do on implementation is add a matching label tag here for the search box

Reply:
Cool I can do that 👍